glue/sample/src/sinter/_collection.py

```diff
@@ -220,10 +219,10 @@ def iter_collect(*,
 
 def collect(*,
             num_workers: int,
-            tasks: Union[Iterator['sinter.Task'], Iterable['sinter.Task']],
+            tasks: Union[Iterator[Task], Iterable[Task]],
```

Comment:
Is this necessary to change? Because these methods are in the external API, I prefer to name the types how users would use them ("sinter.Task" instead of just "Task").